internal/obsreportconfig/obsmetrics/obsmetrics.go

```diff
@@ -19,4 +19,10 @@ package obsmetrics // import "go.opentelemetry.io/collector/internal/obsreportco
 
 const (
 	NameSep = "/"
+
+	MeterName = "io.opentelemetry.collector"
```

Comment:
Why this name? Do we have the same name for receivers/exporters? How does this affect the metrics exposed to Prometheus? /cc @dashpole 

In order to be a no-op change between OC -> Otel for the moment, I think we should have an empty name for the moment. Then when we have the final list of the metrics, we should agree on what scope name we use.

Reply:
This can be removed since I'm no longer using the global meter and using the `ReceiverCreateSettings.MeterProvider` instead, there I'm using the same name as the trace name which is `cfg.ReceiverID.String()`.


> In order to be a no-op change between OC -> Otel for the moment, I think we should have an empty name for the moment.

I think this does not matter for the prometheus exporter,  as the otel scope is ignored now and will be added as additional attributes and metrics per https://github.com/open-telemetry/opentelemetry-specification/pull/2703